.github/actions/analyze/action.yml

```diff
@@ -27,6 +27,9 @@ inputs:
   DATADOG_SESSION_SAMPLE_RATE:
```

Comment:
I guess it would be a good idea to move this to shared-action library cause it seems like it is usable in different projects 🤔

Reply:
https://github.com/deriv-com/shared-actions

Reply:
This is a good idea Niloofar. I think right now, it's very specific to this projects structure.

Also, since our new projects are using vite and not this monorepo type structure, it should be easy to use existing libraries:
https://github.com/marketplace/actions/bundle-size-diff

But in the future we could pull this out, add some more options. are use it elsewhere.

Reply:
Sounds like a plan 😃